VDUBStep/Classes/VDUBStore.h

```diff
@@ -13,9 +13,8 @@
 + (VDUBStore *)sharedInstance;
 
 + (NSManagedObjectContext *)mainContext;
++ (NSManagedObjectContext *)backgroundContext;
 
 - (void)setupCoreDataStackWithURL:(NSURL *)storeURL;
 
-- (NSManagedObjectContext *)mainContext;
```

Comment:
Wondering if I should go ahead and make setupCoreDataStack class-level and remove sharedInstance from the public interface.